src/ui/layouts/common/layouts/AuthenticatedLayout/AuthenticatedSidebar/SideFooter/index.tsx

```diff
@@ -2,31 +2,21 @@ import React from 'react';
 import { MenuItem } from '../Menu/MenuItem';
 import { MenuItemExternal } from './MenuItemExternal';
 import { routePaths } from '../../../../../../../routes/routePaths';
-import { Box, Separator, icons } from '../../../../../../components';
-import { iconSizes, iconColors } from '../../../../../../../constants';
+import { icons } from '../../../../../../components';
+import { iconSizes } from '../../../../../../../constants';
 
 export const SideFooter: React.FC = () => {
   return (
     <>
-      <Box marginHorizontal="md" paddingBottom="md">
-        <Separator.LightNew />
-      </Box>
-
-      <div style={{ marginBottom: '-11px', marginLeft: '-3.5px' }}>
-        <MenuItemExternal
-          id="report"
-          Icon={() => (
-            <icons.supportAgent color={iconColors.white} size={iconSizes.md} />
-          )}
-          to="https://github.com/zenml-io/zenml-dashboard/issues/new/choose"
-          text="Report Issue"
-        />
-      </div>
+      <MenuItemExternal
+        id="report"
+        Icon={() => <icons.supportAgent size={iconSizes.md} />}
+        to="https://github.com/zenml-io/zenml-dashboard/issues/new/choose"
```

Comment:
Just suggestion,
It's look like const URL route. If we can move URL like these Inside const URL routes file. IT will be easier to maintain.